packages/@react-types/checkbox/src/index.d.ts

```diff
@@ -46,7 +42,14 @@ export interface ToggleProps extends InputBase, Validation<boolean>, FocusablePr
   /**
    * Handler that is called when the element's selection state changes.
    */
-  onChange?: (isSelected: boolean) => void,
+  onChange?: (isSelected: boolean) => void
+}
+
+export interface ToggleProps extends ToggleStateOptions, Validation, FocusableProps {
```

Comment:
It looks like `boolean` type argument of `Validation` was removed unintentionally here.
Readded in https://github.com/adobe/react-spectrum/pull/7347